Godeps/_workspace/src/gopkg.in/fatih/set.v0/set_ts.go

```diff
@@ -10,7 +10,7 @@ type Set struct {
 	l sync.RWMutex // we name it because we don't want to expose it
 }
 
-// New creates and initialize a new Set. It's accept a variable number of
+// New creates and initialize a new Set. It accepts a variable number of
```

Comment:
Please revert this change. This is a vendored library from an upstream repo. We don't make modifications to it.